presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergTableProperties.java

```diff
@@ -74,13 +77,22 @@ public IcebergTableProperties(IcebergConfig icebergConfig)
                 .add(stringProperty(
                         FORMAT_VERSION,
                         "Format version for the table",
-                        null,
+                        DEFAULT_FORMAT_VERSION,
```

Comment:
Changed default format version to 2 to follow Iceberg's implementation (https://github.com/apache/iceberg/blob/main/core/src/main/java/org/apache/iceberg/TableMetadata.java#L53C20-L53C48).